### DISCUSSION TITLE
require Claim from array of group

### DISCUSSION THREAD

**Discussion by vinivasundharan** [OpenIDC/mod_auth_openidc#749] [Q&A]
Apache server on CentOS. 
My id_token returns an array of groups where the user a member is. How can I add in the Require claim the group name? ADFS is self hosted on premise. 

id_token

   "group": [
      "Domänen-Benutzer",
      "ais_Build-Deployment-Infrastruktur"
    ],

in conf

     Require claim group:[ais_Build-Deployment-Infrastruktur] also tried without brackets. 

How can i read values from array in apache config?

**Comment by zandbelt** [comment]
I think:
```
 Require claim "group:ais_Build-Deployment-Infrastruktur"
```
should work
